jaypeasee/rancid-tomatillos
# Feature: Routers

# Pull Request Template

#### What’s this PR do?
This PR implements and tests routers on the page. There are now specific Routers for:
* Home (`/`)
* MovieView (`/movie-review/:id`)


#### Where should the reviewer start?
1. Start in `App.js` and review how `Movies` and `Card` now renders and passes data through `<Link />` and `match`.
1. Review `MovieView` to see how that new Route works.
1. Review `Error` to see the catch all for 404 errors.


#### How should this be manually tested?
1. To see the tests passing, run `npm test`.
1. To manually test the routers, run `npm start` to launch the page, enter invalid paths, go to valid paths and refresh. Also use the back and forward buttons to see how everything reacts on the page.

#### Any background context you want to provide? 
This completes iteration #4 of the project.


#### What are the relevant tickets? 
https://app.asana.com/0/1199396273595458/1199530662582697
https://app.asana.com/0/1199396273595458/1199566698816741
